tests/unit/channel_tests.py

```diff
@@ -181,7 +188,7 @@ def test_basic_cancel_calls_raise_if_not_open(self, raise_if_not_open):
 
         self.obj.basic_cancel(consumer_tag, callback=callback_mock)
 
-        raise_if_not_open.assert_called_once()
+        raise_if_not_open.assert_called_once_with()
```

Comment:
The builtin unittest on python 3.5 doesn't have `assert_called_once()`. (earlier and later versions do, though)